packages/dashboards/addon/templates/components/dashboard-action-list.hbs

```diff
@@ -65,7 +65,7 @@
       classNames='action delete'
       model=dashboard
       warnMsg='Are you sure you want to delete this dashboard and all of its widgets?'
-      deleteAction=(route-action 'deleteDashboard' dashboard)
+      deleteAction=(item-action 'DELETE_ITEM' dashboard)
```

Comment:
I feel like we need a more descriptive name for these type of actions instead of just `item-action`.